vm/systemSmartContracts/esdt.go

```diff
@@ -1848,6 +1864,61 @@ func (e *esdt) unSetSpecialRole(args *vmcommon.ContractCallInput) vmcommon.Retur
 	return vmcommon.Ok
 }
 
+func (e *esdt) sendNewTransferRoleAddressToSystemAccount(token []byte, address []byte) {
+	if !e.flagSendTransferRoleAddress.IsSet() {
+		return
+	}
+
+	esdtTransferData := vmcommon.BuiltInFunctionESDTTransferRoleAddAddress + "@" + hex.EncodeToString(token) + "@" + hex.EncodeToString(address)
+	e.eei.SendGlobalSettingToAll(e.eSDTSCAddress, []byte(esdtTransferData))
+}
+
+func (e *esdt) deleteTransferRoleAddressFromSystemAccount(token []byte, address []byte) {
+	if !e.flagSendTransferRoleAddress.IsSet() {
+		return
+	}
+
+	esdtTransferData := vmcommon.BuiltInFunctionESDTTransferRoleDeleteAddress + "@" + hex.EncodeToString(token) + "@" + hex.EncodeToString(address)
+	e.eei.SendGlobalSettingToAll(e.eSDTSCAddress, []byte(esdtTransferData))
+}
+
+func (e *esdt) sendAllTransferRoleAddresses(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
+	if !e.flagSendTransferRoleAddress.IsSet() {
+		e.eei.AddReturnMessage("invalid method to call")
+		return vmcommon.FunctionNotFound
+	}
+	if len(args.Arguments) != 1 {
+		e.eei.AddReturnMessage("too many arguments")
```

Comment:
or could have been the message `wrong number of arguments, expected 1` because we can call this builtin function with 0 parameters